### PR TITLE
Fix timeouts in browser tests

### PR DESCRIPTION
**Pull request by zeljkofilipin**
Jenkins jobs time out because `return jQuery.active` runs for 30
minutes. This commit adds a 5 second time out.

Also added `.DS_Store` (file created by Mac OS) to `.gitignore`.

[Bug: T128065](http://phabricator.wikimedia.org/T128065)

### CODE REVIEW

**Comment by zeljkofilipin** [comment]
cc @Jonas-WMDE @JanZerebecki @hashar

File: tests/browser/features/support/modules/entity_module.rb [hunk 1]
```diff
@@ -112,7 +112,9 @@ def wait_for_mw_notification_shown
   end
 
   def ajax_wait
-    sleep(1.0 / 3) while execute_script('return jQuery.active') != 0
+    Timeout.timeout(5) do
+      sleep(1.0 / 3) while execute_script('return jQuery.active') != 0
```
Comment (thiemowmde) [inline tests/browser/features/support/modules/entity_module.rb]: This tried to run this line of code every 333ms seconds for … **forever**?

**Comment by thiemowmde** [comment]
+2, I would say.

File: tests/browser/features/support/modules/entity_module.rb [hunk 2]
```diff
@@ -112,7 +112,9 @@ def wait_for_mw_notification_shown
...
+    end
     sleep 1
```
Comment (thiemowmde) [inline tests/browser/features/support/modules/entity_module.rb]: Why does it wait for an other second **after** the check succeeded?

**Comment by thiemowmde** [comment]
:+1:

File: tests/browser/features/support/modules/entity_module.rb (hunk 1 above)
Comment (zeljkofilipin) [inline tests/browser/features/support/modules/entity_module.rb]: It did not try, it was running forever. :) Luckily, both Sauce Labs and Jenkins have timeouts, so jobs did timeout eventually.

File: tests/browser/features/support/modules/entity_module.rb (hunk 2 above)
Comment (zeljkofilipin) [inline tests/browser/features/support/modules/entity_module.rb]: I do not know. I did not try to cleanup anything. I just wanted to get rid of the code that caused timeouts.

**Comment by hashar** [comment]
+1

**Comment by WMDE-Fisch** [comment]
As I can recall it this is used in some places to make sure Ajax requests are finished before checking conditions. The places were this is used could be refactored in the future to e.g. use `when_visible` or else.

Nevertheless introducing a timeout here totally makes sense.

**Comment by zeljkofilipin** [comment]
The commit is tested here:

https://integration.wikimedia.org/ci/job/browsertests-Wikidata-WikidataTests-linux-chrome-sauce-T128065/2/consoleFull

It is already catching things that time out:

    ...
    00:25:58.636   Scenario Outline: Save label, description and aliases           # features/header.feature:69
    ...
    00:27:21.103       execution expired (Timeout::Error)
    ...

**Pull request merged**